Civi/Api4/Service/Spec/Provider/MailingGetSpecProvider.php

```diff
@@ -220,19 +220,16 @@ public static function countMailingEvents(array $field): string {
    * @return string
    */
   public static function countIntendedRecipients(array $field): string {
-    $queue = \CRM_Mailing_Event_BAO_MailingEventQueue::getTableName();
-    $mailing = \CRM_Mailing_BAO_Mailing::getTableName();
-    $job = \CRM_Mailing_BAO_MailingJob::getTableName();
```

Comment:
Note that as of https://github.com/civicrm/civicrm-core/pull/29366 these functions do **not** return localized table names, so the before/after of this PR will be the same.